ehsm_kms_service/enroll_app/rest_utils.h

```diff
@@ -30,16 +30,100 @@
  */
 
 
-#ifndef _REST_UTILS_H
-#define _REST_UTILS_H
+#ifndef _ENROLL_APP_REST_UTILS_H
+#define _ENROLL_APP_REST_UTILS_H
 
 #include <stdio.h>
 #include <curl/curl.h>
 #include <iostream>
+#include <jsoncpp/json/json.h>
 
+class RetJsonObj_KMS{
```

Comment:
duplicate with https://github.com/intel/ehsm/blob/main/core/App/json_utils.h?

Reply:
It will be move to include.